File: DrAppAPI/Models/DrProfile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DrAppAPI.Models
{
    public class DrProfile
    {
        public int id_doc { get; set; }
        public Nullable<int> Id_User { get; set; }
        public string name { get; set; }
        public string phone { get; set; }
        public string email { get; set; }
        public string specialty { get; set; }
    }
}

File: app/src/main/java/com/comp313/models/DrProfile.java
package com.comp313.models;

import android.databinding.BaseObservable;

/*
 * By: SHAFIQ-UR-REHMAN
 * Purpose: Model class used for data-binding when reading or writing to db
 */
public class DrProfile extends BaseObservable
{
    public int  id_doc;
    public int     Id_User = 0;
    public String  name;
    public String  phone;
    public String  email;
    public String  specialty;
}
